tasks.py

```diff
@@ -19,6 +20,11 @@
     '==': operator.eq,
 }
 
+
+if not hasattr(inspect, 'getargspec'):
+    inspect.getargspec = inspect.getfullargspec
```

Comment:
is this being tested on windows in any of the workflows? (ie. do any of them run `invoke`)

Reply:
Yes, actually the workflows are being run using `invoke <task-name>` And if they are not this way we probably have to change it to be like this. There are some exceptions such as make docs and so on but other than that should be run using `invoke` (and the current workflows do).